### ISSUE TITLE
Request: Option to display all pot/crate/beehive checks with a special model if unchecked

### ISSUE THREAD

**Issue by Buuhuu42** [rrealmuto/OoT-Randomizer#36]
Hello, hope this is the correct location for this kind of request.

I would like to request an option where all pot / crate / beehive checks are replaced with one of the current "match content" models, if the check is still unchecked. With the amount of checks added, it could be quite overwhelming, so I think this would help get into this option a little bit better. This way it's also easier to see, if a check was already done and also which pots or crates are actual checks.

To summarise, I could imagine to expand the current "match content" / texture options like that:

Replace Pot/Crate/Beehive Models:
- Off
- Texture (Match Contents)
- Texture (All uncollected)

**Comment by rrealmuto** [comment]
Thought this was a neat idea. It's done. Not for beehives though because those don't have a "texture matches contents" setting.
